about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Microverse</title>
+    <link rel="stylesheet" href="./styles/about/about.css">
+    <link rel="shortcut icon" href="./images/microverse.png" type="image/x-icon">
+</head>
+<body>
+    <main class="container">
+        <header class="container__header">
+            <nav class="hamburger-menu">
```

Comment:
- [ ] Remember the navbar on desktop size according the template. 🐦
![image](https://user-images.githubusercontent.com/20563252/136879633-a33180ff-3e94-4ca6-92cb-1581929c76fd.png)